plugin-build/src/test/kotlin/io/sentry/android/gradle/BaseSentryPluginTest.kt

```diff
@@ -94,18 +94,15 @@ abstract class BaseSentryPluginTest(
             // The supported Gradle version can be found here:
             // https://developer.android.com/studio/releases/gradle-plugin#updating-gradle
             // The pair is [AGP Version, Gradle Version]
-            arrayOf("7.0.4", "7.0.2"),
-            arrayOf("7.0.4", "7.1.1"),
             arrayOf("7.0.4", "7.2"),
             arrayOf("7.1.3", "7.2"),
-            arrayOf("7.1.3", "7.3.3"),
-            arrayOf("7.1.3", "7.4"),
-            arrayOf("7.1.3", "7.5"),
             arrayOf("7.2.1", "7.3.3"),
             arrayOf("7.2.1", "7.4"),
             arrayOf("7.2.1", "7.5"),
-            arrayOf("7.3.0-beta05", "7.5"),
-            arrayOf("7.4.0-alpha03", "7.5")
+            arrayOf("7.3.0", "7.4"),
+            arrayOf("7.3.0", "7.5"),
```

Comment:
Should we change `7.5` to `7.5.1` for gradle?

Reply:
they don't do breaking api/behavior changes in patch versions, so we should be good here I think